docs/cookbook/migrating-from-2.2-to-2.3+.md

```diff
@@ -75,3 +75,12 @@ The produced SQLite dump is compatible with the StarDump tool and can be used to
 
 **Note**: the tools are in alpha stage and crashes are expected. Please use the [StarDump](https://github.com/Starcounter/StarDump) and [StarDump.Migrator.2.2](https://github.com/Starcounter/StarDump.Migrator.2.2) repositories to report any issue found.
 
+## Migrating From Starcounter 2.3.1 to 2.3.2 troubleshooting
+
+During migration from Starcounter 2.3.1 to version 2.3.2 you can experience errors about missmatching schemas. Those are aftermaths of changes in data-model schema that were made during 2.3.1 version development. To fix those errors you can manually call run those commands in your database SQL command line:
```

Comment:
> manually call run

Is that a typo? Shouldn't it be either "call" or "run"?

> database SQL command line

What do you mean with this? The administrator?

Reply:
Yeah, that was a typo.